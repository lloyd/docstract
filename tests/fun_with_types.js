/**
 * This is an array of strings
 * @type {array of strings}
 */
exports.arrayOfStrings = [ "foo", "bar", "baz" ];

/**
 * This is an another array of strings
 * @type {strings[]}
 */
exports.arrayOfStrings2 = [ "foo", "bar", "baz" ];

/**
 * This is a *third* array of strings
 * @type { Strings [ ] }
 */
exports.arrayOfStrings3 = [ "foo", "bar", "baz" ];
