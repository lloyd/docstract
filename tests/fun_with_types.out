{
  "filename": "tests/fun_with_types.js", 
  "module": "fun_with_types", 
  "properties": [
    {
      "desc": "This is an array of strings", 
      "name": "arrayOfStrings", 
      "source_lines": [
        1, 
        4
      ], 
      "type": "array of strings"
    }, 
    {
      "desc": "This is an another array of strings", 
      "name": "arrayOfStrings2", 
      "source_lines": [
        7, 
        10
      ], 
      "type": "strings[]"
    }, 
    {
      "desc": "This is a *third* array of strings", 
      "name": "arrayOfStrings3", 
      "source_lines": [
        13, 
        16
      ], 
      "type": "Strings [ ]"
    }
  ]
}
